src/localnet.ts

```diff
@@ -0,0 +1,117 @@
+const { execSync } = require('child_process');
+const path = require('path'); 
+const fs = require('fs');
+
+/**
+ * Runs CLI commands
+ * @param {*} command String command to run
+ */
+const runCommand = (command: string) => {
+  try {
+    // runs given command and prints its output to console
+    execSync(`${command}`, { stdio: 'inherit' });
+  } catch (error) {
+    console.error('Failed to run command: ', error);
+    return false;
+  }
+  return true;
+};
```

Comment:
I saw that we have the same function in [src/create.ts](https://github.com/lambdaclass/zksync-cli/blob/main/src/create.ts#L4-L17). Would it be a good idea to abstract it?